lib/parser/runner.js

```diff
@@ -1,7 +1,11 @@
 var stack = require("./stack");
 
 /**
- *
+ * Compile SQL string from tokens
+ * @param {Array} tokens    set of SQL fragment tokens to be compiled
+ * @param {Object} opts     options for compilation [prettyErrors]
+ * @param {Object} ctx      context to run compilation functions
+ * @returns {Object} output sql and arguments
```

Comment:
Might be able to do better here

    @returns {Object} output.sql
    @returns {Object} output.args

Unsure if thats supported same as [@param](http://stackoverflow.com/questions/6460604/how-to-describe-object-arguments-in-jsdoc)